nmdc_server/api.py

```diff
@@ -552,3 +552,28 @@ async def submit_metadata(
     db.add(submission)
     db.commit()
     return submission
+
+
+@router.get(
+    "/users", responses=login_required_responses, response_model=List[schemas.User], tags=["user"]
+)
+async def get_users(
+    db: Session = Depends(get_db),
+    user: models.User = Depends(admin_required),
+):
+    users = db.query(User).all()
```

Comment:
@subdavis FYI - given the volume of users in not high, we're fetching all rows at once as of now. Would you recommend introducing pagination parameters here?

Reply:
Yeah, I think server-side pagination is a good idea here.   I need to add it to the submission list as well.